Documentation/spec/triple_slash_comments_spec.md

```diff
@@ -0,0 +1,48 @@
+Triple-slash (///) Code Comments Support
+==========================================
+DocFX extracts [triple-slash (///) code comments](https://docs.microsoft.com/en-us/dotnet/articles/csharp/programming-guide/xmldoc/xml-documentation-comments) from .NET source code when running `docfx metadata`. [Tags](https://docs.microsoft.com/en-us/dotnet/articles/csharp/programming-guide/xmldoc/recommended-tags-for-documentation-comments) in triple-slash (///) code comments are converted to corresponding metadata in .NET data model.
+
+> [!NOTE]
+> `docfx` supports [DocFX Flavored Markdown syntax](docfx_flavored_markdown.md) inside triple-slash (///) code comments.
+
+Supported tags
+--------
+### Top level block tags
+Top level block tags are transformed to corresponding metadata in .NET data model.
+
+| Tags            | Metadata name      | Type
+| ---             | ---                | ---
+| `summary`      | `summary`| `string`
+| `remarks`      | `remarks`| `string`
+| `returns`      | `returns` | `string`
+| `value`      | `returns` | `string`
+| `exception`      | `exception`| `List<`@"Microsoft.DocAsCode.DataContracts.ManagedReference.ExceptionInfo"`>`
+| `seealso`      | `seealso`| `List<`@"Microsoft.DocAsCode.DataContracts.ManagedReference.LinkInfo"`>`
```

Comment:
our code also implements top level `see`? although i have no idea how we should show it in the page